test/check-application

```diff
@@ -73,6 +73,24 @@ class TestNavigator(testlib.MachineCase):
         b.click(f"#create-link-{filetype}")
         b.click("button.pf-m-primary")
 
+    # TODO: use https://github.com/cockpit-project/cockpit/pull/19890 when landed
```

Comment:
That  happened now. I triggered a lib-update run, so let's land #217 before this.

Reply:
Landed, so please rebase away.

Reply:
Thanks!